Sources/Sentry/SentrySDK.m

```diff
@@ -62,6 +63,11 @@ + (void)initialize
         sentrySDKappStartMeasurementLock = [[NSObject alloc] init];
         startInvocations = 0;
         _detectedStartUpCrash = NO;
+
+        // The system monitor api is a data bag for system info
+        // that we need for every event, and because this is static info
+        // we need to start it as soon as possible.
+        sentrycrashcm_system_getAPI()->setEnabled(YES);
```

Comment:
`m`: I'm unsure if calling this in `initialize` isn't a bit too early. We only need it when inializing the hub. I think it would be better to call this before enriching the scope. We can call this code multiple times, as when it's initialized it's almost a NoOp. Furthermore, I think we should add a test that fails if we don't call this method in enrich scope, because if we don't the damage is big.